payment_payfip/migrations/11.0.22.02.25/post-migration-reduce-cron-interval.py

```diff
@@ -0,0 +1,17 @@
+import logging
+from openupgradelib import openupgrade
+
+_logger = logging.getLogger(__name__)
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    """Reduce interval of already existing cron (which are in no update mode)."""
+    if not version:
+        return
+
+    cron = env.ref('payment_payfip.cron_check_draft_payment_transactions')
+    if cron.interval_number == 1 and cron.interval_type == "days":
+        cron.interval_number = 5
+        cron.interval_type = "minutes"
```

Comment:
Au cas ou le cron n'existe pas on ne veux pas planter et puis un write pour faire un truc clean ^^. Et j'ai remplacer les " par des ' qui sont pour les textes display en front selon nos conventions
```suggestion
    cron = env.ref('payment_payfip.cron_check_draft_payment_transactions', raise_if_not_found=False)
    if cron and cron.interval_number == 1 and cron.interval_type == 'days':
        cron.write({
            'interval_number': 5,
            'interval_type': 'minutes',
        })
```

Reply:
appliqué dans le nouveau push